README.md

```diff
@@ -151,11 +151,19 @@ Default: false
 
 Puts the less files into the map instead of referencing them.
 
+#### globalVars
```

Comment:
README.md is built by grunt from sources in the build directory. You need to update those, not the main README.